.github/workflows/sv_pipeline_docker.yml

```diff
@@ -155,30 +155,34 @@ jobs:
       DOCKERS_GCP: "./inputs/values/dockers.json"
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           # See the comment on build_args_job.
           fetch-depth: 0
           # Authenticates git using the bot's access token.
           token: ${{ secrets.BOT_PAT }}
 
       - name: Setup Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Azure login
-        uses: azure/docker-login@v1
+        uses: azure/docker-login@v2
         with:
           login-server: ${{ secrets.AZ_CR }}
           username: ${{ secrets.AZ_USERNAME }}
           password: ${{ secrets.AZ_PASSWORD }}
 
+      - name: GCP login
+        uses: google-github-actions/auth@v2.0.0
+        with:
+          credentials_json: '${{ secrets.GCP_SA_CREDENTIALS }}'
```

Comment:
These are already defined somewhere?

Reply:
Yes, these are defined as environment secrets in the repository's configuration.